test/run

```diff
@@ -11,11 +11,8 @@ TEST_SCENARIO=${TEST_SCENARIO:-verify}
 TEST_OS_DEFAULT=$(PYTHONPATH=bots python3 -c 'from machine.machine_core.constants import TEST_OS_DEFAULT; print(TEST_OS_DEFAULT)')
 
 case $TEST_SCENARIO in
-    verify|firefox)
-        # do a development build on the latest Fedora image so that we can spot React and other dev-only warnings
-        if [ "$TEST_OS" = "$TEST_OS_DEFAULT" ]; then
-            export NODE_ENV=development
-        fi
+    verify|devel|firefox|devel+firefox)
+        [ "${TEST_SCENARIO}" = "${TEST_SCENARIO##devel}" ] || export NODE_ENV=development
         [ "${TEST_SCENARIO}" = "${TEST_SCENARIO%%firefox}" ] || export TEST_BROWSER=firefox
```

Comment:
FTR, you can be a bit more liberal here, and use `${TEST_SCENARIO##*devel}` to support any ordering, or extend this to a third scenario in the future. But not important right now.